<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDay</class>
 <widget class="QMainWindow" name="SelectDay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="dayInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Day 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day 7</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Select Day to View:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewReportButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>View Report</string>
    </property>
   </widget>
   <widget class="QTableView" name="itemTable">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="reportLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Item Report</string>
    </property>
   </widget>
   <widget class="QLabel" name="reportLabel_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Member Report</string>
    </property>
   </widget>
   <widget class="QTableView" name="memberTable">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>viewReportButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectDay</receiver>
   <slot>viewReportClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>viewReportClicked()</slot>
 </slots>
</ui>
